packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -0,0 +1,249 @@
+<oui-header data-heading="{{:: 'pci_database_add_title' | translate }}">
+    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
+        <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+            <span data-translate="pci_project_guides_header_all_guides"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+</oui-header>
+
+<div class="mt-4" id="addMessages">
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+</div>
+
+<p>
+    <span data-translate="pci_database_add_description1"></span>
+    <a
+        data-ng-href="{{:: $ctrl.addPrivateNetworksLink }}"
+        data-translate="pci_database_add_description2"
+        data-ng-click="$ctrl.trackDatabases('top_create_private_network')"
+        rel="noopener"
```

Comment:
No `target=_"blank"`

Reply:
This is a link within the manager and would not open in a new window